core/templates/pages/exploration-player-page/learner-experience/tutor-card.component.spec.ts

```diff
@@ -224,6 +224,49 @@ describe('Tutor card component', () => {
     expect(componentInstance.getInputResponsePairId).toHaveBeenCalled();
   }));
 
+  it(
+    'should be able to set celebration component flag to based on diagnostic' +
```

Comment:
I don't understand this test name. Could you explain it more?

Reply:
Removed